README.md

```diff
@@ -371,6 +371,7 @@ Alternatively, you can provide values for each individual meta data (or combinat
 - `javac ... -AfileComment=foo`
 - `javac ... -AfileIndent=foo`
 - `javac ... -AprefixEnclosingClassNames=foo`
+- `javac ... -AinheritComponentAnnotations=false`
```

Comment:
I tried setting up a meta data file, but couldn't get it running with Maven. Maybe you could add an example for how it should be used? (Though probably not as part of this merge request.)

Also, more of a general idea: It would be very useful if these things could also be declared per record class, so probably as part of the `@RecordBuilder` annotation or via a specific annotation for configuration. (Also, not something for this merge request.)

Reply:
I've been thinking about that. Immutables has the `@Value.Style` annotation. Maybe something like that can be done.

Reply:
I like that idea.